docs/UserGuide.md

```diff
@@ -137,6 +135,25 @@ Examples:
 * `find n/alex s/yishun y/sec 3` searches for all students who match all of `n/alex`, `s/yishun` and `y/sec 3`.
 * 
 
+### Recording questions from a student: `question`
+
+(written by: Ying Gao)
+Adds, resolves or remove questions from a specified student.
+
+Format: `question INDEX [a/QUESTION_ADD] [s/QUESTION_INDEX] [d/QUESTION_INDEX]`
+
+* Exactly one of the optional fields must be present.
+* The index and question index **must be positive integers** 1, 2, 3, …​
+* The `a/` prefix adds a new unanswered question to the student at the specified `INDEX`.
+* The `s/` prefix marks an unanswered question, of the student at the specified `INDEX`, at the specified 
+`QUESTION_INDEX` as solved.
+* The `d/` prefix deletes the question at the specified `QUESTION_INDEX` from the student at the specified `INDEX`.
```

Comment:
Minor suggestion, could replace `prefix` with `field` instead, so that a non-technical user reading the user guide can better understand.

Eg: `The s/ field marks an unanswered...`